test/unit/mdc-menu-surface/mdc-menu-surface.test.js

```diff
@@ -45,12 +45,28 @@ function getFixture(open, fixedPosition = false) {
   `;
 }
 
-function setupTest(open = false, fixedPosition = false) {
+function getAnchorFixture(menuFixture) {
+  const anchorEl = bel`
+    <div class="mdc-menu-surface--anchor">
+      <button class="mdc-button">Open</button>
+    </div>
+  `;
+  anchorEl.appendChild(menuFixture);
+  return anchorEl;
+}
+
+function setupTest(open = false, fixedPosition = false, withAnchor = false) {
   const root = getFixture(open, fixedPosition);
   const MockFoundationConstructor = td.constructor(MDCMenuSurfaceFoundation);
   const mockFoundation = new MockFoundationConstructor();
-  const component = new MDCMenuSurface(root, mockFoundation);
-  return {root, component, mockFoundation};
+
+  const fixtures = {root, mockFoundation};
+  if (withAnchor) {
+    fixtures.anchor = getAnchorFixture(root);
+  }
+
+  fixtures.component = new MDCMenuSurface(root, mockFoundation);
```

Comment:
It might be a little easier to read if we have a single object literal containing all properties. E.g.:

```js
const anchor = withAnchor ? getAnchorFixture(root) : undefined;
const component = new MDCMenuSurface(root, mockFoundation);
return {root, mockFoundation, anchor, component};
```